### PR TITLE
Remove unnecessary string building

### PR DESCRIPTION
**Pull request by brfrn169**
Please take a look!

### CODE REVIEW

File: core/src/main/java/com/scalar/db/io/BigIntColumn.java [hunk 1]
```diff
@@ -33,9 +31,9 @@ private BigIntColumn(String name) {
 
   private BigIntColumn(String name, long value, boolean hasNullValue) {
     this.name = Objects.requireNonNull(name);
-    checkArgument(
-        value >= MIN_VALUE && value <= MAX_VALUE,
-        "Out of range column value for BigInt. value: " + value);
```
Comment (brfrn169) [inline core/src/main/java/com/scalar/db/io/BigIntColumn.java:38]: This string building always happened in spite of `IllegalArgumentException` thrown or not.

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Review by komamitsu — APPROVED** [review]
LGTM! 👍

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Review by jnmt — APPROVED** [review]
LGTM! Thank you for the swift improvement!

**Pull request merged**